src/objects/SkinnedMesh.js

```diff
@@ -177,3 +177,45 @@ THREE.SkinnedMesh.prototype = Object.assign( Object.create( THREE.Mesh.prototype
 	}
 
 } );
+
+THREE.SkinnedMesh.prototype.boneTransform = ( function() {
+
+	var clone = new THREE.Vector3(), result = new THREE.Vector3(), skinIndices = new THREE.Vector4(), skinWeights = new THREE.Vector4();
+	var temp = new THREE.Vector3(), tempMatrix = new THREE.Matrix4(), properties = [ 'x', 'y', 'z', 'w' ];
+
+	return function( vertex, index ) {
+
+		if ( this.geometry instanceof THREE.BufferGeometry ) {
+
+			var index4 = index * 4;
+			skinIndices.fromArray( this.geometry.attributes.skinIndex.array, index4 );
+			skinWeights.fromArray( this.geometry.attributes.skinWeight.array, index4 );
+
+		} else if ( this.geometry instanceof THREE.Geometry ) {
+
+			skinIndices.copy( this.geometry.skinIndices[ index ] );
+			skinWeights.copy( this.geometry.skinWeights[ index ] );
+
+		}
+
+		var clone = vertex.clone().applyMatrix4( this.bindMatrix ); result.set( 0, 0, 0 );
```

Comment:
We shouldn't `clone()` in this method, it is likely to be called often. I would suggest modifying the input vertex instead, and perhaps changing the signature:

```typescript
transformVertex ( index : number, target : THREE.Vector3 ) : this 
```